yaml2ncml/yaml2ncml.py

```diff
@@ -224,14 +182,42 @@ def add_aggregation_scan(text, a):
           bed_thickness='sediment_bed_thickness')
 
 
+def _compose(attrs, name):
+    return OrderedDict(
+        ('_'.join([name, key]), attrs[name].get(key))
+        for key in attrs[name].keys() if attrs[name].get(key)
+    )
+
+
+templ = Template("""<?xml version="1.0" encoding="UTF-8"?>
+<netcdf xmlns="http://www.unidata.ucar.edu/namespaces/netcdf/ncml-2.2">
+{% for name, value in global_attrs | dictsort %}  <attribute name="{{ name }}" type="String" value="{% if value is string %}{{ value }}{% else %}{{ value|join(',') }}{%endif%}"/>
+{% endfor %}
+""")  # noqa
```

Comment:
This is the template. It is basically a for-loop that adds the `name`, `value` for the many global attributes.